AutoDMG/IEDCLIController.py

```diff
@@ -240,6 +247,7 @@ def addargsBuild_(self, argparser):
         argparser.add_argument(u"-o", u"--output", help=u"DMG output path")
         argparser.add_argument(u"-i", u"--installer", help=u"Override installer in template")
         argparser.add_argument(u"-n", u"--name", help=u"Installed system volume name")
+        argparser.add_argument(u"-s", u"--size", help=u"Installed system volume size, in GB")
```

Comment:
You could enforce an int argument with type=int here instead, and let argparse deal with the error.